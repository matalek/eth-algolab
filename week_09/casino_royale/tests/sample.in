4
4 4 2
2 3 1
0 1 1
0 2 1
1 3 1
6 4 2
0 3 1
1 5 1
3 4 1
0 3 1
6 4 2
0 3 32
1 5 40
3 4 16
0 3 38
5 10 2
0 2 25
1 3 36
0 4 100
3 4 40
1 3 25
2 3 14
3 4 21
1 3 92
3 4 58
0 1 51
