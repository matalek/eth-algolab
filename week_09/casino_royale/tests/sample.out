4
3
94
301
